src/Compilers/CSharp/Portable/Binder/MethodArgumentInfo.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I couldn't tell what's actually getting changed. Are BOMs getting normalized or something like that?

Reply:
Yea, it may be a little easier to see with git diff

```
diff --git a/src/Compilers/CSharp/Portable/Binder/MethodArgumentInfo.cs b/src/Compilers/CSharp/Portable/Binder/MethodArgumentInfo.cs
index 5e742d84d72..b8a2f76c02e 100644
--- a/src/Compilers/CSharp/Portable/Binder/MethodArgumentInfo.cs
+++ b/src/Compilers/CSharp/Portable/Binder/MethodArgumentInfo.cs
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+<EF><BB><BF>// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
```

Reply:
Our .editorconfig specifies utf8-bom as the charset. These changes add the necessary BOM.